templates/zerver/create_realm.html

```diff
@@ -6,7 +6,32 @@
 {% block portico_content %}
 <script type="text/javascript">
 $(function () {
+    var timer;
+
     common.autofocus('#email');
+
+    $('#email').on("keydown", function () {
+        clearTimeout(timer);
+    });
+    $('#email').on("keyup", function () {
+        clearTimeout(timer);
+        timer = setTimeout(checkRealm, 1000, $('#email').val());
+    });
+
+    function checkRealm (email_value) {
+        $.ajax({
+          url: "/json/check_realm/" + email_value,
```

Comment:
Why is the thing you're checking the `email_value`?  We want to check if the subdomain is available, not validate the email, here.

Reply:
Yes, I agree. But I am removing the other div which was meant for showing the errors for the email input. I am sort of merging these two into one. That's why I need to validate the email too.

Reply:
Otherwise, there will be two divs, both showing error messages. Which isn't pretty, in my opinion.